docs/admin/access.rst

```diff
@@ -116,17 +122,15 @@ or by using user management on the project page, as described in :ref:`manage-ac
 Automatic group assignments
 ---------------------------
 
-You can set up Weblate to automatically add users to groups based on their
-e-mail addresses. This automatic assignment happens only at the time of account creation.
-
-This can be set up in the Django admin interface for each group (in the
-:guilabel:`Authentication` section).
+From the :guilabel:`Authentication` in the Django admin interface,
+users can be assigned to groups [you want this for] automatically based
```

Comment:
Why angle brackets here?

Reply:
@orangesunny ?

Reply:
Well, I felt that it’s appropriate to use them in that case, but can be changed.

Reply:
[ ] are called *square brackets*

*Angle brackets* are < >

Reply:
Well, from many sources, [EnglishClub](https://www.englishclub.com/writing/punctuation-brackets.htm) included, round brackets seems right.

Reply:
Learnt something there. :)

Reply:
Me too! So are you gonna change the brackets, or will we keep them?